packages/jest-transform/src/__tests__/script_transformer.test.js

```diff
@@ -541,6 +535,94 @@ describe('ScriptTransformer', () => {
     expect(writeFileAtomic.sync).toHaveBeenCalledTimes(1);
   });
 
+  it('should write a source map for the instrumented file when transformed', () => {
+    config = {
+      ...config,
+      transform: [['^.+\\.js$', 'preprocessor-with-sourcemaps']],
+    };
+    const scriptTransformer = new ScriptTransformer(config);
+
+    const map = {
+      mappings: ';AAAA',
+      version: 3,
+    };
+
+    // A map from the original source to the instrumented output
+    /* eslint-disable sort-keys */
+    const instrumentedCodeMap = {
+      version: 3,
+      sources: ['banana.js'],
+      names: ['content'],
+      mappings: ';;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;AAAAA,OAAO',
+      sourcesContent: ['content'],
+    };
+    /* eslint-enable sort-keys */
```

Comment:
```suggestion
    /* eslint-enable */
```

Reply:
Taking this suggestion (and the other one)